src/Command/AbstractClientRequestCommand.php

```diff
@@ -47,8 +47,11 @@ abstract class AbstractClientRequestCommand extends Command
 
     protected function configure(): void
     {
-        // General option to get a raw result. Can be used for further processing.
-        $this->addOption('raw', 'r', InputOption::VALUE_OPTIONAL, 'Return result as raw object (e.g. json)', false);
+        $this
+            // General option to get a raw result. Can be used for further processing.
+            ->addOption('raw', 'r', InputOption::VALUE_OPTIONAL, 'Return result as raw object (e.g. json)', false)
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Outputs the operations but will not execute anything')
+        ;
```

Comment:
Regarding consistency: This should **not** go into an extra line. See the other commands.